it/uk/gov/hmrc/soletraderidentificationfrontend/controllers/PersonalDetailsControllerISpec.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.soletraderidentificationfrontend.controllers
+
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.api.test.Helpers._
+import uk.gov.hmrc.soletraderidentificationfrontend.assets.MessageLookup.{PersonalDetails => messages}
+import uk.gov.hmrc.soletraderidentificationfrontend.utils.{ComponentSpecHelper, ViewSpec}
+
+class PersonalDetailsControllerISpec extends ComponentSpecHelper with ViewSpec {
+
+  "GET /personal-details-page" should {
+    lazy val result = get("/personal-details-page")
+    lazy val doc: Document = Jsoup.parse(result.body)
+
+    "return OK" in {
+      result must have(httpStatus(OK))
+    }
+
+    "have a view with the correct title" in {
+      doc.title mustBe messages.title
+    }
+
+    "have a view with the correct heading" in {
+      println(doc)
+      doc.getH1Element.text mustBe messages.heading
+    }
+  }
+}
```

Comment:
Can we have a test for the submit route please? It's fine that it just returns NotImplemented.